modules/animation/include/modules/animation/datastructures/valuekeyframesequence.h

```diff
@@ -69,6 +70,16 @@ struct DefaultInterpolationCreator {
     }
 };
 
+template <typename T>
```

Comment:
Seems like a useful concept in other places. Do we want to put these in, for example, util/concepts.h?